plugins/elementspath/plugin.js

```diff
@@ -149,6 +149,13 @@
 					case 32: // SPACE
 						onClick( elementIndex );
 						return false;
+					case CKEDITOR.ALT + 121: // ALT + F10 (#438).
+						editor.execCommand( 'toolbarFocus' );
+						return false;
+					case CKEDITOR.ALT + 122: // ALT + F11 (#438).
```

Comment:
There is already case for focusing editor. Additionally I'm not sure if we really need it, as <kbd>Esc</kbd> already handles it.